webapp/channels/src/components/admin_console/custom_enable_disable_guest_accounts_setting.tsx

```diff
@@ -19,8 +19,16 @@ type Props = {
     showConfirm: boolean;
 }
 
-export default class CustomEnableDisableGuestAccountsSetting extends React.PureComponent<Props> {
-    public handleChange = (id: string, value: boolean, submit = false) => {
+const CustomEnableDisableGuestAccountsSetting: React.FC<Props> = ({
```

Comment:
Just for consistency with most of the rest of the code, there is no need to be explicit with the React.FC type. We can leave it something like `const foo = ({...}: Props) {`
```suggestion
const CustomEnableDisableGuestAccountsSetting = ({
```